sdk/eventhub/azure-eventhub/stress/test-resources.json

```diff
@@ -0,0 +1,169 @@
+{
```

Comment:
You can if you want, but technically this doesn't need to be checked in, as the scripts will build the bicep file on the fly. I had it checked in for service bus because it pre-dated this functionality.

Reply:
I just copied and pasted our EH test-resrouces.json, we don't have a bicep in our repo yet.

I'll still unclear on what's difference between bicep/json.

What I'm understanding is that they are almost the same thing defining the resources to be created, while bicep is more powerful (with nice syntax sugar?)

Can we say that ultimately we will just have bicep files, all json files will be gone?

Reply:
Correct. Bicep compiles to and de-compiles from ARM templates. Beyond syntactic sugar it also has some useful features like module imports and good error messages.

We are moving towards a goal state of having all json files be gone. If you want to start by copying over test-resources.json that's fine, though you may have to remove some template parameters that won't be provided today by the script. @ckairen is working on migrating deployment to use the common New-TestResources.ps1 script though which would fix this.